pkg/api/2019-04-30/types.go

```diff
@@ -65,6 +65,9 @@ type Properties struct {
 
 	// AuthProfile (in): configures OpenShift authentication
 	AuthProfile *AuthProfile `json:"authProfile,omitempty"`
+
+	// MonitorProfile configures which log analytics workspace to use.
+	MonitorProfile *MonitorProfile `json:"monitorProfile,omitempty"`
```

Comment:
Is it agreed with Anubhuti that we are changing the existing external API?

Reply:
(as opposed to creating a new external API version...)

Reply:
cc @amanohar

Reply:
This will need to be a new API version. 

 (I also added it in the doc): Do we want `MonitorProfile` to be opinionated towards Log Analytics constructs? Or should it be more like authProfile, i.e. generic enough to support other Log providers? 

Note that its important to think of API model not just in context of `3.11` but also future ARO in general (and `4.x`) and options it will give to customers without big changes to the API model in the future.